src/main/java/core/basesyntax/service/impl/DataParserImpl.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataParser;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataParserImpl implements DataParser {
+    private static final String SEPARATOR = ",";
+    private static final int INFORMATION_LINE_INDEX = 0;
+    private static final int OPERATION_INDEX = 0;
+    private static final int FRUIT_NAME_INDEX = 1;
+    private static final int FRUIT_QUANTITY_INDEX = 2;
+
+    @Override
+    public List<FruitTransaction> parse(List<String> lines) {
+        lines.remove(INFORMATION_LINE_INDEX);

```

Comment:
Directly removing an element from the input list (lines) can be dangerous since it alters the original data which might not be an expected behavior. Consider creating a new list or skipping the first line during the parsing process instead of modifying the input list.